### PR TITLE
[13.0][IMP] queue_job: check that queue_job_notify trigger exists in db

### PR DESCRIPTION
**Pull request by nilshamerlinck**
Could save time in cases such as https://github.com/OCA/queue/issues/262 :)

### CODE REVIEW

**Review by lmignon — APPROVED** [review]

File: queue_job/jobrunner/runner.py [hunk 1]
```diff
@@ -278,12 +278,28 @@ def _has_queue_job(self):
...
+                return False
+            cr.execute(
+                """SELECT COUNT(1)
+                FROM information_schema.triggers
+                WHERE event_object_table = %s
+                AND trigger_name = %s""",
+                ("queue_job", "queue_job_notify"),
+            )
+            if cr.fetchone()[0] != 3:  # INSERT, DELETE, UPDATE
+                _logger.debug(
```
Comment (guewen) [inline queue_job/jobrunner/runner.py]: Since having the module installed without the trigger is never normal, should we log with warn or error level?

Comment (lmignon) [reply queue_job/jobrunner/runner.py]: Good catch.... IMO we should log with error level.

Comment (nilshamerlinck) [reply queue_job/jobrunner/runner.py]: makes sense! fixed

**Review by guewen — APPROVED** [review]
Thanks!

**Review by sbidoul — APPROVED** [review]
LGTM. Good idea, thanks!

**Pull request merged**